Apps/W1/Shopify/app/src/Companies/Codeunits/ShpfyCompanyExport.Codeunit.al

```diff
@@ -142,6 +145,12 @@ codeunit 30284 "Shpfy Company Export"
 
         CompanyLocation."Phone No." := Customer."Phone No.";
 
+        ShpfyTaxRegistrationIdMapping := Shop."Shpfy Comp. Tax Id Mapping";
+        CompanyLocation."Tax Registration Id" := ShpfyTaxRegistrationIdMapping.GetTaxRegistrationId(Customer);
```

Comment:
possible overflow. Should the interface instead return Text[150] if this is where it's going to be used?

Reply:
limited return value to 150